src/plugins/data_source_management/public/components/utils.ts

```diff
@@ -181,6 +182,17 @@ export async function getDataSources(savedObjectsClient: SavedObjectsClientContr
     );
 }
 
+export async function getDataConnections(savedObjectsClient: SavedObjectsClientContract) {
+  return savedObjectsClient
+    .find<DataConnectionSavedObjectAttributes>({
+      type: 'data-connection',
+      perPage: 10000,
+    })
+    .then((response) => {
+      return response?.savedObjects;
+    });
```

Comment:
given this is new, can we cover with tests?

Reply:
added tests